src/components/tiles/WeatherForecast.tsx

```diff
@@ -1,21 +1,150 @@
 import { useQuery } from "@tanstack/react-query";
 import axios from "axios";
+
+import { Card } from "react-bootstrap";
+import { TbSun, TbCloudRain, TbCloudSnow } from "react-icons/tb";
 import ErrorAlert from "../alerts/ErrorAlert";
+
 import Tile from "../Tile";
 import LoadingTile from "./LoadingTile";
 
+const curr = new Date();
+const today = `${curr.getDate()}.${curr.getMonth() + 1}.${curr.getFullYear()}`;
+const tomorrow = `${curr.getDate() + 1}.${
+  curr.getMonth() + 1
+}.${curr.getFullYear()}`;
+const dat = `${curr.getDate() + 2}.${
+  curr.getMonth() + 1
+}.${curr.getFullYear()}`;
+
```

Comment:
Das kann man so direkt nicht machen. Angenommen `today` ist der `31.01.2023` dann wäre bei dir `tomorrow` der `32.01.2023`. Kann man bspw [so ](https://www.tutorialrepublic.com/faq/how-to-add-days-to-current-date-in-javascript.php)lösen